doc/zephyr.doxyfile.in

```diff
@@ -1179,7 +1184,7 @@ SOURCE_BROWSER         = NO
 # documentation.
 # The default value is: NO.
 
-INLINE_SOURCES         = NO
+INLINE_SOURCES         = YES
```

Comment:
I assume this was activated to include the test function bodies into the doxygen documentation am I right?

Reply:
dont remember tbh, I assume yes.